test/integration.test.js

```diff
@@ -10,22 +10,80 @@ const FastifySecretsHashiCorp = require('..')
 const SECRET_KEY = 'exampleKey'
 const SECRET_NAME = uuid.v4()
 const SECRET_CONTENT = uuid.v4()
-const MOUNT_POINT = 'fastify-integration-test'
+const MOUNT_POINT = {
+  V1: 'fastify-integration-test_v1',
+  V2: 'fastify-integration-test_v2'
+}
+const VERSIONS = {
+  V1: 1,
+  V2: 2
+}
 const vault = nodeVault()
 
-async function setup() {
-  await vault.mount({ mount_point: MOUNT_POINT, type: 'kv', options: { version: 1 } })
-  await vault.write(`${MOUNT_POINT}/${SECRET_NAME}`, { [SECRET_KEY]: SECRET_CONTENT })
+async function setup(version) {
+  await vault.mount({ mount_point: MOUNT_POINT[version], type: 'kv', options: { version: VERSIONS[version] } })
+
+  let path = `${MOUNT_POINT[version]}/${SECRET_NAME}`
+  let data = { [SECRET_KEY]: SECRET_CONTENT }
+
+  if (version === 'V2') {
+    path = `${MOUNT_POINT[version]}/data/${SECRET_NAME}`
+    data = { data: { [SECRET_KEY]: SECRET_CONTENT } }
+  }
```

Comment:
I would improve a bit this test file, I found it hard to read because there are some mapping objects (`MOUNT_POINT`) and conditions (`version === 'V2'`). 
Choosing one style would improve the reading. Do you agree?